Tools/GoogleTestAdapter.FxCop

```diff
@@ -337,17 +347,2518 @@
      </Message>
     </Messages>
    </Namespace>
+   <Namespace Name="XamlGeneratedNamespace">
+    <Messages>
+     <Message TypeName="DeclareTypesInMicrosoftOrSystemNamespace" Category="Microsoft.MSInternal" CheckId="CA904" Created="2017-09-15 23:25:59Z">
+      <Issue>
+       <Item>XamlGeneratedNamespace</Item>
+      </Issue>
+     </Message>
+    </Messages>
+   </Namespace>
   </Namespaces>
+  <Targets>
```

Comment:
Does it make sense to have all elements below, which are easily to generate by running FxCop, checked into repo?

Reply:
Removed